trio/_core/tests/test_run.py

```diff
@@ -1709,59 +1693,25 @@ async def main():
 
         _core.run(main)
 
-    class Deferred:
-        "Just kidding"
+    for bad_call in bad_call_run, bad_call_spawn:
 
-    with ignore_coroutine_never_awaited_warnings():
-        for bad_call in bad_call_run, bad_call_spawn:
+        async def f():  # pragma: no cover
+            pass
 
-            async def f():  # pragma: no cover
-                pass
+        with ignore_coroutine_never_awaited_warnings():
```

Comment:
You don't need this anymore since you added the logic to close the passed coroutine, right?

Reply:
Yep! you're right